src/authorizations/components/AllAccessTokenOverlay.tsx

```diff
@@ -58,6 +59,10 @@ const AllAccessTokenOverlay: FC<OwnProps> = props => {
   )
 
   const handleSave = () => {
+    if (!meID) {
```

Comment:
meID being empty is what's causing the error mentioned in the bug ticket to occur. Checking for its value, then dispatching an action to store result in state will make sure we are including `meID` when making a post request to create a token.